app/src/main/java/io/islnd/android/islnd/app/loader/LoaderId.java

```diff
@@ -0,0 +1,12 @@
+package io.islnd.android.islnd.app.loader;
+
+public class LoaderId {
+    public static final int COMMENT_LOADER_ID = 0;
```

Comment:
This is nice to move all the loader IDs into one class :+1: 
We might want to distinguish between the loaders that are classes and the activities that implement the loader callbacks. For example change `EDIT_PROFILE_LOADER_ID` to `EDIT_PROFILE_ACTIVITY_LOADER_ID`, but leave the IDs like `COMMENT_LOADER_ID` that are actual classes.
I think this might be slightly clearer and easier to maintain, but it is not a strong concern.